### ISSUE TITLE
Determining neighbors of a leaf?

### ISSUE THREAD

**Issue by ArjunNarayanan** [rdeits/RegionTrees.jl#30]
Is there a way to determine all the neighbors of a given leaf? If this requires a pull request I am happy to try and look into it :)

**Comment by rdeits** [comment]
You could do `children(parents(x))` to get all of the cells which are children of the same parent. Would that work for your case?

**Comment by ArjunNarayanan** [comment]
No, that will not suffice for me, I need to determine all of the neighbors of a cell. I figured I could construct this information top down -- starting from the root and then updating the neighbors of the children at each level.

**Comment by ashwanirathee** [comment]
Could this be a default feature to get neighbours quickly: children(parents(x))?
Also @ArjunNarayanan when talking about all neigbours of a cell, did you mean all blocks nearby in nd space?
